Sources/Public/Animation/LottieAnimationView.swift

```diff
@@ -103,7 +103,8 @@ open class LottieAnimationView: LottieAnimationViewBase {
     textProvider: AnimationKeypathTextProvider = DefaultTextProvider(),
     fontProvider: AnimationFontProvider = DefaultFontProvider(),
     configuration: LottieConfiguration = .shared,
-    logger: LottieLogger = .shared)
+    logger: LottieLogger = .shared,
+    animateLayoutChangesWithCurrentCoreAnimationContext: Bool = true)
```

Comment:
Rather than adding this as an argument to `init`, can we just expose the `public var` for it? A good example of this pattern is the `mainThreadRenderingEngineShouldForceDisplayUpdateOnEachFrame` property added in https://github.com/airbnb/lottie-ios/pull/2168